src/pages/trader-tools/pnl-for-multipliers/_pnl_multipliers_calculator.tsx

```diff
@@ -1689,13 +1689,13 @@ const PnlMultipliersCalculator = () => {
                                             'Let’s say you want to calculate the take profit amount when you want to open a position for Volatility 100 Index priced at 3376.24 USD with a stake amount of 10 USD, a multiplier value of x100, and a take profit level of 3400 in Up direction.',
                                         )}
                                     </Text>
-                                    <Show.Desktop max_width="mobileL">
+                                    <Show.Desktop max_width="tablet">
```

Comment:
Can we also refactor this with the new `show` component under `components/containers/visibility.tsx`

Reply:
Can't find `Show` component in visibility.tsx, maybe you meant components/containers/show.js? Isn't it already using the show component with Show.Desktop?

Reply:
@konstantinosG-deriv , import Desktop from visibility.tsx and replace <Show.Desktop max_width="tablet"> with <Desktop breakpoint="tablet">.Similarly for Mobile as well.

Reply:
@vijayasree-deriv  Oh okay 👍